srtcore/congctl.cpp

```diff
@@ -58,7 +58,7 @@ void SrtCongestion::Check()
 class LiveCC: public SrtCongestionControlBase
 {
     int64_t  m_llSndMaxBW;          //Max bandwidth (bytes/sec)
-    int      m_iSndAvgPayloadSize;  //Average Payload Size of packets to xmit
+    size_t   m_iSndAvgPayloadSize;  //Average Payload Size of packets to xmit
```

Comment:
The below field has also `z` type identifier in the name, which distinguishes it from `i` signed integer type. I think the name should be also changed accordingly.